src/main/java/uk/gov/hmcts/reform/blobrouter/services/BlobVerifier.java

```diff
@@ -44,31 +45,33 @@ public VerificationResult verifyZip(String blobName, byte[] rawBlob) {
             return ok();
         } catch (DocSignatureFailureException ex) {
             logger.info("Invalid signature. Blob name: {}", blobName, ex);
-            return error("Invalid signature");
+            return error(ErrorCode.ERR_SIG_VERIFY_FAILED, "Invalid signature");
         } catch (InvalidZipArchiveException ex) {
             logger.info("Invalid zip archive. Blob name: {}", blobName, ex);
-            return error("Invalid zip archive");
+            return error(ErrorCode.ERR_ZIP_PROCESSING_FAILED, "Invalid zip archive");
```

Comment:
Doesn't `ERR_ZIP_PROCESSING_FAILED` represent generic failure to process a file? Shouldn't it be `FILE_VALIDATION_FAILURE` instead?

Reply:
we don't have the suggested error code `FILE_VALIDATION_FAILURE`. 
https://github.com/hmcts/reform-scan-notification-service/blob/cfd3645203223639a74b6e4d4e6b26689582a700/src/main/java/uk/gov/hmcts/reform/notificationservice/model/common/ErrorCode.java#L12

Reply:
Sorry, I was thinking about events. All good.